### DISCUSSION TITLE
Port to json-c

### DISCUSSION THREAD

**Discussion by ondrejholy** [FreeRDP/FreeRDP#10009] [General]
Would you accept a port from `cjson` to `json-c`? I am asking because everything else uses the `json-c` project in RHEL for this purpose.

**Comment by akallabeth** [comment]
@ondrejholy no preference there.
since `CJSON` is already done I´d vote for implementing this as alternative first.

**Reply by ondrejholy** [reply]
Ok, thanks, I will try to do so next month if all goes as expected.
